docs: make quickstart match the v1.5 UI (#4164)
akuity/kargo: docs/docs/20-quickstart/index.md

configured to do so automatically, and in fact, the branches referenced by their
`targetRevision` fields do not even exist yet.

> ![Argo Dashboard](img/argo-dashboard.png)

Reviewer: ```suggestion
![Argo CD Dashboard](img/argo-dashboard.png)
```

      This will take you to a list of `Project`s.  It currently includes only
      the one created in the previous step.

>       ![Kargo Projects Dashboard](img/kargo-projects.png)

Reviewer: ```suggestion
      ![Kargo Projects List](img/kargo-projects.png)
```

        * Three `Stage`s representing distinct instances of our demo
          application.

>         ![Kargo Project Dashboard](img/kargo-dashboard-projects.png)

Reviewer: ```suggestion
        ![Kargo Project View](img/kargo-dashboard-projects.png)
```

## Your First Promotion

> 1. To promote `Freight` to the `test` `Stage`, click the truck icon next
>    to <Hlt>test</Hlt> and then select <Hlt>Promote</Hlt>:

Reviewer: ```suggestion
--------------------------------------------------------------------------------
1. To promote `Freight` to the `test` `Stage`, click the truck icon in the
   header of that node and then select <Hlt>Promote</Hlt>:
```

    ![Kargo Promote Option](img/kargo-promote-option.png)

>     Next, <Hlt>Select</Hlt> the `Freight` from the freight timeline and confirm
>     the promotion by selecting <Hlt>Promote</Hlt>:

Reviewer: ```suggestion
    Next, click the <Hlt>Select</Hlt> button on the `Freight` in the timeline
    that you'd like to promote (there's just the one right now), then confirm
    this action by clicking <Hlt>Promote</Hlt>:
```

## Your First Promotion

1. To promote `Freight` to the `test` `Stage`, click the truck icon in the
   header of that node and then select <Hlt>Promote</Hlt>:

    ![Kargo Promote Option](img/kargo-promote-option.png)

    Next, click the <Hlt>Select</Hlt> button on the `Freight` in the timeline
    that you'd like to promote (there's just the one right now):

    ![Kargo Promote Confirm](img/kargo-promote-confirm.png)

    Confirm this action by clicking <Hlt>Promote</Hlt>:

    ![Kargo Promote Confirm](img/kargo-promote-option-2.png)

>     Once the promotion process completes successfully, the <Hlt>Promotion</Hlt> view will display
>     the status as <Hlt>Succeeded</Hlt>, along with details about the promotion process, including
>     each step that was executed:

Reviewer: ```suggestion
    A summary of the promotion will pop up and will be updated in real-time as
    the steps of the promotion process complete. Once they have all completed,
    promotion's status will change to <Hlt>Succeeded</Hlt>.
```

   header of that node and then select <Hlt>Promote</Hlt>:

    ![Kargo Promote Option](img/kargo-promote-option.png)

1. From the timeline at the top of the screen, select the `Freight` you'd like
   to promote into the `test` `Stage` by clicking <Hlt>Select</Hlt>:

   ![Kargo Promote Confirm](img/kargo-promote-confirm.png)

1. Confirm the action by clicking <Hlt>Promote</Hlt>:

   ![Kargo Promote Confirm](img/kargo-promote-option-2.png)

   A summary of the `Promotion` will pop up and will be updated in real-time as
   the steps of the promotion process complete. Once they have all completed,
   the `Promotion`'s status will change to <Hlt>Succeeded</Hlt>:

   ![Kargo Promotion View](img/kargo-promotion-view.png)

   You will also notice the freight timeline has been automatically updated.
   Every piece of `Freight` in the timeline is color-coded to indicate which
   `Stage`s (if any) are actively using it. You will see the one piece of
   `Freight` currently in the timeline is marked with the same color as the
   `test` `Stage`'s node in the pipeline. This indicates this piece of `Freight`
   is currently used by that `Stage`.

## After Your First Promotion

> Following your first Kargo will periodically execute a health check on the `test`
> `Stage`. After the first successful health check, the `test` `Stage`'s healthy
> state will be reflected by a heart icon on its node in the pipeline.

Reviewer: ```suggestion
Following your first promotion, Kargo will periodically execute a health check
on the `test` `Stage`. After the first successful health check, the `test`
`Stage`'s healthy state will be reflected by a heart icon on its node in the
pipeline.
```

## Your First Promotion

> 1. To promote `Freight` to the `test` `Stage`, click the truck-arrow icon next

Reviewer: ```suggestion
1. To promote `Freight` to the `test` `Stage`, click the truck icon next
```

   to <Hlt>test</Hlt> and then select <Hlt>Promote</Hlt>:

    ![Kargo-Promote](img/kargo-promote-option.png)

    Next, <Hlt>Select</Hlt> the `Freight` from the <Hlt>Freight Timeline</Hlt> and confirm
    the promotion by selecting <Hlt>Promote</Hlt>:

    ![Kargo-Promote](img/kargo-promote-option-2.png)

    Once the promotion process completes successfully, the <Hlt>status</Hlt> field in

>     the <Hlt>Promotion</Hlt> tab of the `Stage` view shows <Hlt>Succeeded</Hlt>:

Reviewer: This text does not match what is depicted in the image.

Reviewer: > the Promotion tab of the Stage view

This is _not_ what is depicted in that image. This does indeed tell you that the Promotion has succeeded, but this is a view of the Promotion itself. It is not a view of the Stage.

The view of the Stage looks different:

![Screenshot 2025-05-21 at 11 45 04 AM](https://github.com/user-attachments/assets/c158f5d1-482b-42e7-8a65-b7c3a9e0ece9)

Also, you've not instructed the reader on how to navigate to the screen above -- not that I'm not suggesting you should _at this step_. The information they need to see is available in the Promotion view they are already looking at.

Just correct the text to more accurately describe what the reader is looking at.

    promotion. It is color-coded to indicate which `Stage`s are actively using
    each piece of `Freight`.

1. Select the external link icon on the <Hlt>test</Hlt> `Stage` to reveal additional details about the `Stage`
   including its status, current `Freight`, and history:

>    ![Kargo-external-link](img/kargo-external-link.png)

Reviewer: I see you revised this line to describe what clicking this button does, but are still using the phrase "external link icon." Please change that.
